tests/test_partner.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+
+import easypost
+
+
+@pytest.mark.vcr()
+@pytest.mark.skip(reason="Do not record this test in cassette because there is sensitive info")
+def test_partner_referral_user_create(prod_api_key):
+
+    created_referral_user = easypost.beta.Partner.create_referral(
+        name="test test",
+        email="test@test.com",
+        phone="8888888888",
+    )
+
+    assert str.startswith(created_referral_user.id, "user_")
```

Comment:
Please add an assertion like all other tests to assert something like the name matches from the response.

Reply:
resolved